hazelcast/src/main/java/com/hazelcast/nio/tcp/nonblocking/NonBlockingSocketReader.java

```diff
@@ -245,6 +245,20 @@ public String toString() {
         return connection + ".socketReader";
     }
 
+    @Override
```

Comment:
Do we need this method at all? Why can't the AbstractHandler take care of this instead of adding more methods to the subclasses?